ios/RNGestureHandler.m

```diff
@@ -236,6 +254,48 @@ - (RNGestureHandlerState)state
     return RNGestureHandlerStateUndetermined;
 }
 
+#pragma mark Manual activation
+
+- (void)stopActivationBlocker
+{
+  if (_manualActivationRecognizer != nil) {
+    [_manualActivationRecognizer fail];
+  }
+}
+
+- (void)setManualActivation
```

Comment:
would be cleaner if you'd make a property for manualActivation in class definition:
```
@property (....) BOOL manualActivation;
```

Then in `configure` you just do:
```
self.manualActivation = [RCTConvert BOOL:prop]
```

And you can still have a "setter":
```
- (void)setManualActivation:(BOOL)manualActivatiobn
{
  _manualActivation = manualActivation;
  // all the logic
}
```

Reply:
Done in https://github.com/j-piasecki/react-native-gesture-handler/pull/2/commits/f4af31172f09f0962dd23a4dc11ef8748bc6f21f